src/Nri/Ui/AssignmentIcon/V1.elm

```diff
@@ -0,0 +1,141 @@
+module Nri.Ui.AssignmentIcon.V1 exposing (diagnostic, guidedDraft, peerReview, practice, quickWrite, quiz, selfReview)
+
+{-|
+
+@docs diagnostic, guidedDraft, peerReview, practice, quickWrite, quiz, selfReview
+
+    import Css
+    import Html.Styled exposing (..)
+    import Html.Styled.Attrbutes exposing (css)
+    import Nri.Ui.AssignmentIcon.V1 as AssignmentIcon
+    import Nri.Ui.Colors.V1 as Colors
+    import Nri.Ui.Svg.V1 as Svg
+
+    view : Html msg
+    view =
+        div
+            [ css [ Css.color Colors.lichen ]
+            ]
+            [ Svg.toHtml AssignmentIcon.diagnostic ]
+
+-}
+
+import Nri.Ui.Svg.V1
+import Svg.Styled as Svg
+import Svg.Styled.Attributes as Attributes
+
+
+{-| -}
+diagnostic : Nri.Ui.Svg.V1.Svg
+diagnostic =
+    Svg.svg
+        [ Attributes.x "0px"
+        , Attributes.y "0px"
```

Comment:
❓ what do the x and y attributes do here?  should they be here?

Reply:
Yeah some of these attributes (these x and y coordinates and the `enable-background: new` stuff) we didn't know why it was there -- it was part of the svg originally. I'll try to remove them and see if I can see a difference. I don't know what they would be doing though.